src/api/alchemy-provider.ts

```diff
@@ -152,8 +155,8 @@ export class AlchemyProvider
   }
 
   /**
-   * Overrides the base {@link providers.JsonRpcProvider.send} method to
-   * implement custom logic for sending requests to Alchemy.
+   * Overrides the base {@link JsonRpcProvider.send} method to implement custom
```

Comment:
Rationale behind keeping some `@link`s like this vs. not using it like for `WebSocketProvider` below?

Reply:
I'd prefer to use `@link`s where applicable since you can go to the original definition directly when using an IDE, but switched it in other places where we're not importing from the top level.